Makefile

```diff
@@ -20,6 +20,12 @@ export OS_OUTPUT_GOPATH
 # mounting secrets specific to a build environment.
 export OS_BUILD_IMAGE_ARGS
 
+# Tests run using `make` are most often run by the CI system, so we are OK to
+# assume the user wants jUnit output and will turn it off if they don't.
+ifndef JUNIT_REPORT
```

Comment:
Isn't `JUNIT_REPORT ?= true` the equivalent of this?  That's what we do for OS_OUTPUT_GOPATH